src/main/java/com/vdenotaris/spring/boot/security/saml/web/config/WebSecurityConfig.java

```diff
@@ -349,7 +349,7 @@ public SingleLogoutProfile logoutprofile() {
     public KeyManager keyManager() {
         DefaultResourceLoader loader = new DefaultResourceLoader();
         Resource storeFile = loader
-            .getResource(env.getProperty("keystorePath"));
```

Comment:
Is this property in any of the template files? If so, can you remove those too?

Reply:
I have no idea why I took that property variable out...